{"uuid":"03ea67da-d3fa-4464-b8c4-d842d9b05ae7","historyId":"9fa04731f320ce3457a0656b9ef34a0f","fullName":"com.bridgelabz.Testcase.LoginUsingXcelData.loginUsingXcel","labels":[{"name":"package","value":"com.bridgelabz.Testcase.LoginUsingXcelData"},{"name":"testClass","value":"com.bridgelabz.Testcase.LoginUsingXcelData"},{"name":"testMethod","value":"loginUsingXcel"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"NewSeleniumProject"},{"name":"subSuite","value":"com.bridgelabz.Testcase.LoginUsingXcelData"},{"name":"host","value":"LAPTOP-2F8N43DG"},{"name":"thread","value":"18564@LAPTOP-2F8N43DG.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"loginUsingXcel","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1","trace":"java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:57)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:803)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:728)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:924)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1\r\n\tat com.bridgelabz.Testcase.LoginUsingXcelData.getTestData(LoginUsingXcelData.java:40)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1708151840860,"stop":1708151840863}